tests/tracer/test_logging.py

```diff
@@ -0,0 +1,518 @@
+import os
```

Comment:
I benchmarked running this test file and it took about 45 seconds. We will run the circleci tracer job 4 times for each python version in our test suite. We run the tracer test suite with [2 attrs and packaging](https://github.com/DataDog/dd-trace-py/blob/v1.1.4/riotfile.py#L235-L236) versions). Since this file will be run 4 times in each tracer job merging these tests will increase the time for these jobs to complete from [~12mins ](https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/18052/workflows/553ef889-bd6f-4e7d-8595-3ba9cb81cc5d/jobs/1226813) to ~[15mins](https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/18102/workflows/5ee77b7f-3080-451d-a3f5-f9afb5bbd74b/jobs/1230022). The total runtime could increase significantly if we decide to test across more package versions.

@brettlangdon @Kyle-Verhoog is this acceptable?

We could move these tests to the [test_logging](https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/18052/workflows/553ef889-bd6f-4e7d-8595-3ba9cb81cc5d/jobs/1226804) test suite instead. Increasing that ci run from ~3mins to ~6mins seems a bit more optimal. We could also remove/refactor a few of the parameterized tests to decrease our total circleci cost. Running this test file could get expensive.

Reply:
@mabdinur thanks for looking into that.

What is the cause of the slowness?

Do we think there are any unnecessary tests in this file?

Reply:
Would it help the speed of the tests to only test a few log file levels?

Reply:
I remove the test for the child logger and right now it takes less than 3 minutes: https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/18176/workflows/a816ef05-c69c-4990-aedb-33af390440cd/jobs/1234804/steps